packages/gestalt-design-tokens/tokens/vr-theme-web-mapping/comp-web-color-light.json

```diff
@@ -316,6 +316,113 @@
             "value": "{sema.color.pressed.background.default.value}"
           }
         }
+      },
```

Comment:
same here, we remove the base files and move the logic to the comp-web ones in light and dark

Reply:
Done